### PR TITLE
add log when creating rocksdb if not exist

### PR DESCRIPTION
**Pull request by noamhammer**
_add log when creating rocksdb if not exist_

---
In addition, remove unneeded `get` method on the graph

### CODE REVIEW

File: packages/dag/src/nodemap.ts [hunk 1]
```diff
@@ -352,7 +352,7 @@ export class DataNodeMap<T> extends AbstractNodeMap {
       const reason = this.has(id) ? 'Node has no data' : 'Node does not exist'
       throw new Error(`Cannot get data of "${id}": ${reason}`)
     }
-    return this.nodeData.get(id) as T
```
Comment (erikzaadi) [inline packages/dag/src/nodemap.ts:355]: How is this related to the log?

File: packages/dag/src/nodemap.ts [hunk 2]
```diff
@@ -352,7 +352,7 @@ export class DataNodeMap<T> extends AbstractNodeMap {
...
+    return result as T
```
Comment (ori-moisis) [inline packages/dag/src/nodemap.ts:355]: how is this related to this PR?
it seems correct, but it should at least be mentioned in the PR title / commit message

**Review by ori-moisis — APPROVED** [review]
I assume you don't want to create a separate PR for the completely unrelated thing, so at least add it to the description and the commit message when you merge?
